### DISCUSSION TITLE
日志一堆应该是报错，不知道是什么。大佬帮看一下

### DISCUSSION THREAD

**Discussion by xwndo** [jxxghp/MoviePilot#371] [问题咨询]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.0.5

### 问题类型

主程序运行问题

### 问题描述

![截图_20230822211557](https://github.com/jxxghp/MoviePilot/assets/44762565/edc9eb30-f252-4dbf-8827-c9054cdbe28e)
[moviepilot.log](https://github.com/jxxghp/MoviePilot/files/12409004/moviepilot.log)

**Comment by developer-wlj** [comment]
启用了代理 代理不可用引起的

**Comment by xwndo** [comment]
> 启用了代理 代理不可用引起的

好的，谢谢，我检查一下

**Comment by xwndo** [comment]
> 启用了代理 代理不可用引起的

我查了下代理正常的情况下也会这样

**Comment by xwndo** [comment]
![截图_20230823105901](https://github.com/jxxghp/MoviePilot/assets/44762565/cf8df626-ad6d-430f-8a88-a71e5e7efb01)
![截图_20230823105913](https://github.com/jxxghp/MoviePilot/assets/44762565/2a792fb1-e8ea-48ce-99f1-18a778de15e1)
